### ISSUE TITLE
Native html "Select" menu - "Select a benchmark"

### ISSUE THREAD

**Issue by amberderosa** [FreedomScientific/standards-support#181]
### Summary

> Brief description of the issue. Please list any specific steps.
> 
Example:

Go to https://investor.vanguard.com/etf/profile/performance/voo
2. Go to "Select Benchmark drop down"

### Expected result

> What did you expect to happen? Screen reader should read expanded state and collapsed state. This works in IE and JAWS but not with Chrome and JAWS. 

### Actual result

> What actually did happen?
Sr is reading only the expanded state not the collapsed state. This works in IE and JAWS but not with Chrome and JAWS. 

### Example

> A test case that demonstrates the issue - 
A test case that demonstrates the issue -
Go to page - https://investor.vanguard.com/etf/profile/performance/voo
get to the Select benchmark drop down. 
close the benchmark drop down 

#### JAWS version and build number

JAWS Version 18.0.1835
 
#### Operating System and version

Windows 10 

#### Browser and version:
Chrome - Version 68.0.3440.75 (Official Build) (64-bit)

**Comment by menovak** [comment]
hi amberderosa,

I found the **Select benchmark drop down.** at the URL provided, and inspecting the source code, this appears as a standard HTML `<select>` element (which, btw is missing an accessible name).  If I test the HTML `<select>` element (e.g., combo box) using JAWS 2018/2019 I found the following.

If I TAB to the HTML `<select>` element and then press the  SPACEBAR key to open/expand, or ESCAPE key to close/collapse the combo box and pay attention to what JAWS announces for this action:

IE11 - announces expanded, usually announces collapsed
FireFox - announces nothing
Chrome - usually announces expanded, seldom announces collapsed

I then loaded up a stadard WAI HTML `<select>` element from here, (https://www.w3.org/WAI/tutorials/forms/grouping/) and retested that `<select>` element under the Grouping items in select elements heading as I had done above and found:

IE11 - announces expanded, usually announces collapsed
FireFox - announces nothing
Chrome - usually announces expanded, seldom announces collapsed

Results:
- for each browser, JAWS treats the `<select>` elements the same 
- for each browser, JAWS does announce 'combo box, to change the selection, use the arrow keys', in other words, a screen reader user does not have to expand the combo box to use it.
- the fact that JAWS provides a bit more information about the combo box in IE11 maybe due to JAWS heuristics or differences in the accessibility API between browsers.  In either case, there does not seem to be an issue with JAWS.

Thanks
Mark
